### DISCUSSION TITLE
the latest commit with goodwe library 0.4.8 is awesome!

### DISCUSSION THREAD

**Discussion by gitboi-99** [mletenay/home-assistant-goodwe-inverter#293] [General]
Finally I can get the meter data for my MS series inverter directly from the inverter via this integration, no need to use SEMS portal with 3min frequency :)

Bit of a pain to update though given there's no new release yet, had to manually overwrite the old python files in the custom integrations directory.

Could we get a new release made for the latest commit please? ^_^

**Comment by BenBarat** [comment]
Belatedly, can I ask for this as well?

**Reply by gitboi-99** [reply]
it actually is there already, I was just downloading something wrong the first time I think. v0.9.9.27 installed via HACS should add in the active power sensor.
